packages/pegasus/test/peg.test.js

```diff
@@ -325,4 +325,4 @@ async function testRemotePeg(t) {
   });
 }
 
-test('remote peg', t => testRemotePeg(t));
+test.failing('remote peg', t => testRemotePeg(t));
```

Comment:
@michaelfig is this functionality still required?

Reply:
these changes are no longer part of this PR, so marking this resolved